packages/valaxy-theme-yun/components/YunToggleLocale.vue

```diff
@@ -19,7 +19,7 @@ const toggleLocales = () => {
 </script>
 
 <template>
-  <button class="yun-icon-btn" :title="t('button.toggle_langs')" style="color:var(--va-c-text)" @click="toggleLocales">
+  <button class="yun-icon-btn" :title="t('button.change_lang')" style="color:var(--va-c-text)" @click="toggleLocales">
```

Comment:
After thinking about it, I think we should use `switch_lang`, because the buttons here can only switch directly between Chinese and English.
Users can continue to expand i18n if more languages ​​become available.

Reply:
What about extensibility in the future?

Reply:
Since `change lang` does not create any misunderstanding I think we can use that directly, consider this might be wide-spreading in the future, updating this entry of existing users might not be worth it IMHO 🤔

Reply:
```yml
switch_lang: xxx
switch_langs: xxx
```

I think we do not need to modify the old content, but just extend the new field.

> I want to be consistent with light/dark by `switch`.

Reply:
Right.

Reply:
Updated